### ISSUE TITLE
[BE] Attachement to email content

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#105]
Dependent on #104 

Frontend offers users the possibility to add attachements to the email content, such as pdfs, jpgs or pngs for example. These attachements are then successfully sent to the TalktixChatAPI.

## Acceptance criteria
- [ ] attachments are send flawless to the TalktixChatAPI
- [ ]  attachments are added to the text input
- [ ]  at leat pdfs and pngs/jpgs can be submitted as attachement

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
